### PR TITLE
Front/header connect back

### PR DESCRIPTION
**Pull request by Yakiigit**
- Can now add game from profile page
- Can now remove game from profile page
- Search bar added in header
- Created GamerAvatar component, still in progress

### CODE REVIEW

**Comment by Yakiigit** [comment]
There's currently a breaking bug on my branch, please do not merge for now

**Comment by Yakiigit** [comment]
Looking good now

File: backend/src/server/createServer.ts [hunk 1]
```diff
@@ -21,9 +22,10 @@ export function createServer() {
   // Backend endpoints
   app.use('/authentication', authenticationRouter);
   app.use('/comment', mustBeAuthenticated, commentRouter);
+  app.use('/games', gameRoutes);
   app.use('/gamers', mustBeAuthenticated, gamerRoutes);
   app.use('/user', mustBeAuthenticated, userInfoRouter);
   app.use('/posts', mustBeAuthenticated, postRoutes);
-  app.use('/postTags', mustBeAuthenticated, postTagRoutes);
+  app.use('/postTags', postTagRoutes);
```
Comment (Xseuguh) [inline backend/src/server/createServer.ts:29]: :( why ?

Comment (Yakiigit) [reply backend/src/server/createServer.ts:29]: I had to remove it to add tags via Postman

Comment (Yakiigit) [reply backend/src/server/createServer.ts:29]: I don't think it is that useful either, there is nothing to do on post tag routes

**Pull request merged**